src/components/column-sorting/column-sorting.component.less

```diff
@@ -0,0 +1,36 @@
+@import "../../styles/variables.less";
+ux-column-sorting{
+    .ux-column-sorting {
+        padding-left: 5px;
+        display: inline-block;
+        width: 36px;
+        height: 28px;
```

Comment:
This is required to keep the number sitting top the top right of the icon as it does in ng1 column sorting.